ttnn/cpp/ttnn/operations/sharding_utilities.hpp

```diff
@@ -91,15 +92,39 @@ ShardingConfig get_specs_for_sharding_partition(
 
 namespace sharded_accessor_utils {
 
+enum class ArgConfig : uint8_t {
+    CTA = 0,
+    RankCRTA = 1 << 0,
+    NumBanksCRTA = 1 << 1,
+    TensorShapeCRTA = 1 << 2,
+    ShardShapeCRTA = 1 << 3,
+    BankCoordsCRTA = 1 << 4,
+    CRTA = RankCRTA | NumBanksCRTA | TensorShapeCRTA | ShardShapeCRTA | BankCoordsCRTA
+};
+
+using ArgsConfig = Flags<ArgConfig>;
+constexpr ArgsConfig operator|(ArgConfig a, ArgConfig b) noexcept { return ArgsConfig(a) | b; }
+constexpr ArgsConfig operator|(ArgConfig a, ArgsConfig b) noexcept { return ArgsConfig(a) | b; }
```

Comment:
```suggestion
constexpr ArgsConfig operator|(ArgConfig a, ArgConfig b) noexcept { return ArgsConfig(a) | b; }
constexpr ArgsConfig operator|(ArgConfig a, ArgsConfig b) noexcept { return ArgsConfig(a) | b; }
```
I don't see it being used.